lib/extensions/auto_satin.py

```diff
@@ -2,7 +2,7 @@
 
 import inkex
 
-from ..elements import SatinColumn, Stroke
+from ..elements import SatinColumn
```

Comment:
Why do you remove the import of Stroke here? It's needed in line 62...

Reply:
Oops, this must have been introduced by some merge issue. Thanks for catching. Should be fixed now.